bindata/v4.1.0/kube-apiserver/kubeconfig-cm.yaml

```diff
@@ -8,7 +8,7 @@ data:
     apiVersion: v1
     clusters:
       - cluster:
-          certificate-authority: /etc/kubernetes/static-pod-resources/configmaps/kube-apiserver-server-ca/ca-bundle.crt
+          certificate-authority: /etc/kubernetes/static-pod-resources/secrets/localhost-recovery-client-token/ca.crt
```

Comment:
this CA must include the localhost-recovery serving cert CA. It's one created by the operator when the localhost-recovery serving cert secret is created. How do we make sure that this is known to kcm when creating the localhost-recovery service account. This smells like a race.

Reply:
I wonder how this can ever work:
- bootstrap control plane is shut down if all control plane pods are up and ready on the master nodes
- the master kcm only takes over from bootstrap kcm when the latter is killed
- the localhost-recovery CA only shows up in the kube-apiserver-server-ca after the bootstrap has been started
- how can the localhost-recovery service-account ever include the localhost-recovery CA as that service-account is need to start a master kube-apiserver.

Reply:
The localhost-recovery-client-token is revisioned. So this might work after bootstrapping is over. But the bootstrapping step is still unclear to me.